app/src/main/java/com/android/sample/ui/createRecipe/ListIngredientScreen.kt

```diff
@@ -0,0 +1,267 @@
+                  clip = true)
+              .background(
+                  MaterialTheme.colorScheme.secondary,
+                  shape = RoundedCornerShape(INGREDIENT_PREVIEW_CORNER.dp))) {
+        Row(modifier = Modifier.fillMaxWidth().padding(SMALL_PADDING.dp)) {
+          // Adds left space
+          Spacer(modifier = Modifier.width(IMAGE_SPACER.dp))
+          Column(
+              verticalArrangement = Arrangement.SpaceBetween,
+              horizontalAlignment = Alignment.Start,
+              modifier =
+                  Modifier.padding(
+                      start = PADDING.dp,
+                      top = PADDING.dp,
+                      bottom = PADDING.dp,
+                      end = (PADDING * 2).dp)) {
+                Text(
+                    text = ingredient.name,
+                    style = MaterialTheme.typography.titleSmall,
+                    color = MaterialTheme.colorScheme.onPrimary,
+                    maxLines = 2,
+                    overflow = TextOverflow.Ellipsis)
+
+                Spacer(modifier = Modifier.height(SMALL_PADDING.dp))
+
+                // Editable quantity field
+                OutlinedTextField(
+                    value = quantity,
+                    onValueChange = { newQuantity ->
+                      quantity = newQuantity
+                      ingredientViewModel.updateQuantity(ingredient, quantity)
+                    },
+                    label = {},
+                    keyboardOptions = KeyboardOptions(imeAction = ImeAction.Next),
+                    keyboardActions = KeyboardActions(onNext = { focusManager.clearFocus() }),
+                    shape = RoundedCornerShape(INGREDIENT_PREVIEW_CORNER.dp),
+                    modifier = Modifier.testTag("recipeNameTextField${ingredient.name}"),
+                    colors =
+                        TextFieldDefaults.colors(
+                            focusedContainerColor = MaterialTheme.colorScheme.background,
+                            unfocusedContainerColor = lightGrayInput,
+
+                            // Make sure these are transparent to avoid unwanted lines
+                            focusedIndicatorColor = Color.Transparent,
+                            disabledIndicatorColor = Color.Transparent,
+                            unfocusedIndicatorColor = Color.Transparent,
+                        ),
+                    textStyle = MaterialTheme.typography.bodyMedium,
+                    maxLines = INPUT_MAX_LINE,
+                )
+              }
+        }
+
+        // Remove button
+        IconButton(
+            onClick = { ingredientViewModel.removeIngredient(ingredient) },
+            modifier =
+                Modifier.align(Alignment.TopEnd)
+                    .padding(end = SMALL_PADDING.dp, top = SMALL_PADDING.dp)) {
+              Icon(
+                  modifier = Modifier.testTag("removeIngredientIcon${ingredient.name}"),
```

Comment:
Similarly here, our goal is to verify that the iconButton is displayed for each ingredient, rather than focusing on the specific text being displayed.